# English (United States) translations for fb_vmware.
# Copyright (C) 2022 Frank Brehm, Berlin
# This file is distributed under the same license as the fb_vmware project.
# Frank Brehm <frank@brehm-online.com>, 2022.
#
msgid ""
msgstr ""
"Project-Id-Version: fb_vmware 0.3.1\n"
"Report-Msgid-Bugs-To: frank@brehm-online.com\n"
"POT-Creation-Date: 2022-05-16 07:43+0200\n"
"PO-Revision-Date: 2022-05-02 09:45+0200\n"
"Last-Translator: Frank Brehm <frank@brehm-online.com>\n"
"Language: en_US\n"
"Language-Team: Frank Brehm <frank@brehm-online.com>\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.10.1\n"

#: lib/fb_vmware/about.py:282 lib/fb_vmware/cluster.py:316 lib/fb_vmware/datastore.py:222
#: lib/fb_vmware/dc.py:161 lib/fb_vmware/disk.py:388 lib/fb_vmware/ds_cluster.py:126
#: lib/fb_vmware/ether.py:435 lib/fb_vmware/host.py:242 lib/fb_vmware/host.py:586
#: lib/fb_vmware/host_port_group.py:198
msgid "The given parameter {p!r} on calling method {m}() has failing attributes"
msgstr ""

#: lib/fb_vmware/about.py:290 lib/fb_vmware/cluster.py:324 lib/fb_vmware/dc.py:169
#: lib/fb_vmware/disk.py:397
msgid "Parameter {t!r} must be a {e} object, a {v} object was given instead."
msgstr ""

#: lib/fb_vmware/about.py:303 lib/fb_vmware/cluster.py:349 lib/fb_vmware/datastore.py:262
#: lib/fb_vmware/dc.py:189 lib/fb_vmware/disk.py:419 lib/fb_vmware/ds_cluster.py:152
#: lib/fb_vmware/ether.py:495 lib/fb_vmware/host.py:274 lib/fb_vmware/host.py:613
#: lib/fb_vmware/host_port_group.py:222
msgid "Creating {} object from:"
msgstr ""

#: lib/fb_vmware/about.py:339 lib/fb_vmware/disk.py:424 lib/fb_vmware/ether.py:500
#: lib/fb_vmware/host_port_group.py:227
msgid "Created {} object:"
msgstr ""

#: lib/fb_vmware/base.py:106
msgid "Invalid port number {!r} for the VSphere server, must be 0 < PORT <= 65536."
msgstr ""

#: lib/fb_vmware/base.py:221
msgid "Connecting to vSphere host {h}:{p} as {u!r} ..."
msgstr "Connecting to vSphere host {h}:{p} as {u!r} …"

#: lib/fb_vmware/base.py:239
msgid "Disconnecting from vSphere host {}."
msgstr ""

#: lib/fb_vmware/cluster.py:263 lib/fb_vmware/datastore.py:349 lib/fb_vmware/dc.py:127
#: lib/fb_vmware/disk.py:274 lib/fb_vmware/ds_cluster.py:191 lib/fb_vmware/ether.py:305
#: lib/fb_vmware/host.py:517 lib/fb_vmware/host_port_group.py:161
msgid "Comparing {} objects ..."
msgstr "Comparing {} objects …"

#: lib/fb_vmware/cluster.py:299 lib/fb_vmware/cluster.py:306
msgid "The given parameter {p!r} on calling method {m}() is not a sequence type."
msgstr ""

#: lib/fb_vmware/cluster.py:357
msgid "Cluster {c!r} has network {n!r}."
msgstr ""

#: lib/fb_vmware/cluster.py:364
msgid "Cluster {c!r} has datastore {d!r}."
msgstr ""

#: lib/fb_vmware/datastore.py:231 lib/fb_vmware/ds_cluster.py:135
msgid "Parameter {t!r} must be a {e}, {v!r} was given."
msgstr ""

#: lib/fb_vmware/datastore.py:367 lib/fb_vmware/ds_cluster.py:209
msgid "Invalid item type {{!r}} to set, only {} allowed."
msgstr ""

#: lib/fb_vmware/datastore.py:369
msgid "The key {k!r} must be equal to the datastore name {n!r}."
msgstr ""

#: lib/fb_vmware/datastore.py:370 lib/fb_vmware/ds_cluster.py:212
msgid "None type as key is not allowed."
msgstr ""

#: lib/fb_vmware/datastore.py:371 lib/fb_vmware/ds_cluster.py:213
msgid "Empty key {!r} is not allowed."
msgstr ""

#: lib/fb_vmware/datastore.py:372 lib/fb_vmware/ds_cluster.py:214
msgid "Object {{!r}} is not a {} object."
msgstr ""

#: lib/fb_vmware/datastore.py:604
msgid "Could not handle datastore type {!r}."
msgstr ""

#: lib/fb_vmware/datastore.py:611
msgid "Could not found a datastore for {c:0.1f} GiB of type {t!r}."
msgstr ""

#: lib/fb_vmware/datastore.py:618
msgid "Searching datastore for {c:0.1f} GiB of type {t!r}."
msgstr ""

#: lib/fb_vmware/disk.py:435 lib/fb_vmware/ether.py:511 lib/fb_vmware/host_port_group.py:238
msgid "Invalid type {t!r} as an item of a {c}, only {o} objects are allowed."
msgstr ""

#: lib/fb_vmware/disk.py:503 lib/fb_vmware/ether.py:579 lib/fb_vmware/host_port_group.py:327
msgid "{m} takes at most {max} arguments ({n} given)."
msgstr ""

#: lib/fb_vmware/disk.py:543
msgid "Disk is not in disk list."
msgstr ""

#: lib/fb_vmware/ds_cluster.py:211
msgid "The key {k!r} must be equal to the datastore cluster name {n!r}."
msgstr ""

#: lib/fb_vmware/errors.py:38
msgid "No VSphere datastores found."
msgstr ""

#: lib/fb_vmware/errors.py:68
msgid "Invalid name {n!r} for a {o} VSphere object."
msgstr ""

#: lib/fb_vmware/errors.py:71
msgid "Invalid name {!r} for a VSphere object."
msgstr ""

#: lib/fb_vmware/errors.py:89
msgid "The VSphere datacenter {!r} is not existing."
msgstr ""

#: lib/fb_vmware/errors.py:105
msgid "The VSphere Virtual machine {!r} was not found."
msgstr ""

#: lib/fb_vmware/errors.py:125
msgid "No SAN based datastore found with at least {m:0.0f} MiB == {g:0.1f} GiB available space found."
msgstr ""

#: lib/fb_vmware/errors.py:143
msgid "The network {!r} is not existing."
msgstr ""

#: lib/fb_vmware/errors.py:162
msgid "Could not connect to the vSphere host {h}:{p} as user {u!r}."
msgstr ""

#: lib/fb_vmware/errors.py:184
msgid "Timeout on creating VM {vm!r} after {to:0.1f} seconds."
msgstr ""

#: lib/fb_vmware/errors.py:187
msgid "Timeout on creating VM {!r}."
msgstr ""

#: lib/fb_vmware/ether.py:444 lib/fb_vmware/host.py:250 lib/fb_vmware/host.py:594
#: lib/fb_vmware/host_port_group.py:206
msgid "Parameter {t!r} must be a {e}, {v!r} ({vt}) was given."
msgstr ""

#: lib/fb_vmware/ether.py:472
msgid "Checking class of ethernet card: {!r}"
msgstr ""

#: lib/fb_vmware/ether.py:618
msgid "card is not in card list."
msgstr ""

#: lib/fb_vmware/host.py:599
msgid "Host {!r} seems to be offline!"
msgstr ""

#: lib/fb_vmware/host_port_group.py:366
msgid "group is not in group list."
msgstr ""

#: lib/fb_vmware/obj.py:78
msgid "The type of a {} may not be None."
msgstr ""

#: lib/fb_vmware/obj.py:83 lib/fb_vmware/obj.py:106 lib/fb_vmware/obj.py:129
msgid "Invalid {w}.{p} {v!r}."
msgstr ""

#: lib/fb_vmware/obj.py:145
msgid "The name prefix of a {} may not be None."
msgstr ""

#: lib/fb_vmware/obj.py:149
msgid "Invalid name prefix {p!r} for a {o}."
msgstr ""

#: lib/fb_vmware/xlate.py:98
msgid "Module directory:"
msgstr ""

#: lib/fb_vmware/xlate.py:99
msgid "Base directory:"
msgstr ""

#: lib/fb_vmware/xlate.py:100
msgid "Locale directory:"
msgstr ""

#: lib/fb_vmware/xlate.py:101
msgid "Locale domain:"
msgstr ""

#: lib/fb_vmware/xlate.py:102
msgid "Found .mo-file:"
msgstr ""

